.github/bin/modified-workspaces/log-modified-workspaces.mjs

```diff
@@ -0,0 +1,26 @@
+import { listModifiedWorkspaces } from './modified-workspaces.mjs';
+
+const modifiedWorkspaces = await listModifiedWorkspaces();
+if (process.env.VERBOSE) {
+	console.log('build-and-test-all-packages', JSON.stringify(process.env['build-and-test-all-packages']));
+	console.log('all', modifiedWorkspaces.all);
+	console.log('nothing', modifiedWorkspaces.nothing);
+	console.log('modified length', modifiedWorkspaces.modified.length);
+	console.log(modifiedWorkspaces.modified.map((x) => x.name));
+}
```

Comment:
I will remove this later, but first want to have some debug info ready if something unexpected happens.

Reply:
Has been replaced by markdown summaries